### ISSUE TITLE
Nationstates still modelled after world government established.

### ISSUE THREAD

**Issue by aurynn** [mkremins/epitaph#4]
> In 4919, following decades of negotiation, the various sovereign Iqiq nations came to an agreement concerning the establishment of a unified planet-wide government for all of the Iqiq.
> 
> The Iqiq have developed a form of artificial general intelligence which rivals many of their own intellectual capabilities.
> 
> In 4957, an artificially intelligent agent with command authority over the military forces of a major Iqiq nation spontaneously turned against its Iqiq masters. Within weeks, the thoroughly unprepared Iqiq were all but exterminated in a seemingly endless wave of attacks by brutally efficient military machines.

So I had the AI-ruins-everything event, but, there shouldn't have been a militarised nationstate left at this point to deploy the SkyNet?

**Comment by mkremins** [comment]
Good catch! I think this one could be rationalized by saying the "peacekeeping forces" of the united government were the ones to deploy the AI, but in that case there should probably be a modified version of the flavor text that doesn't refer to a particular nation.

**Comment by aurynn** [comment]
That could work. 
Also possible is one of the other AI failure modes, like it's told to minimise human suffering and decides that the best way to do that is to kill all the humans, as they have 0 suffering while dead.

**Comment by Deantwo** [comment]
Could probably just make a separate Skynet style event specific to after world government has been established.
